src/commands/junit/api.ts

```diff
@@ -10,6 +10,8 @@ import {renderUpload} from './renderer'
 
 import {CI_JOB_URL, CI_PIPELINE_URL, GIT_SHA} from '../../helpers/tags'
 
+const getSafeFileName = (unsafeFileName: string) => unsafeFileName.replace(/[^a-z0-9]/gi, '_')
```

Comment:
Can we have a test of this function? Also, it'd worth it to have a comment explaining why this change in the name is required. (Are there some requirements in the name of the file for S3)?

Reply:
done